jenkins/milestone-deploy/Jenkinsfile

```diff
@@ -0,0 +1,61 @@
+pipeline {
+  agent any
+
+  options {
+    buildDiscarder(logRotator(numToKeepStr:'15'))
+    disableConcurrentBuilds()
+    timeout(time: 120, unit: 'MINUTES')
+  }
+
+  parameters {
+    choice(name: 'RELEASE_TYPE', choices: ['M0', 'M1', 'M2', 'M3', 'RC1', 'RC2', 'GA'], description:
+      '''
+        Kind of milestone release to build.
+      ''')
+  }
+
+  environment {
+    DOWNLOAD_AREA = '/home/data/httpd/download.eclipse.org/modeling/tmf/xtext'
+    REPOSITORY_PATH="${DOWNLOAD_AREA}/updates/milestones"
+  }
+
+  tools {
+    maven "apache-maven-3.8.6"
+    jdk "temurin-jdk17-latest"
```

Comment:
do we intentionally not run with java 11 (and 202203 tp)

Reply:
I thought that it would be better to build against the same TP that is part of simrel.
Do you think it's better to use the minimal TP also when releasing?

Reply:
i assume this is again a thig of developer discipline. @szarnekow what do you think

Reply:
I'm ok with it for M0. I'd like to review the involved shell-scripts together with the Jenkinsfile in more details though.